### PR TITLE
[INLONG-10034][Manager] Support calling the audit service interface to query audit information

### LINKED ISSUE #10034: [Improve][Manager] Support calling the audit service interface to query audit information
### Description

Support calling the audit service interface to query audit information.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #10034

### Motivation

Support calling the audit service interface to query audit information.

### Modifications

Support calling the audit service interface to query audit information.

### CODE REVIEW

**Comment by dockerzhang** [comment]
@doleyzi PTAL，thanks

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            } else {
+                auditSet = getAuditInfoList(request, request.getInlongGroupId(), request.getInlongStreamId(), auditId);
+            }
+            auditVOList.add(new AuditVO(auditId, auditName, auditSet, auditIdMap.getOrDefault(auditId, null)));
+            this.latch.countDown();
+        } catch (Exception e) {
+            LOGGER.error("query audit failed for request={}", request);
+            this.latch.countDown();
+            throw new BusinessException("query audit failed");
+        }
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java:101]:
```suggestion
        } catch (Exception e) {
            LOGGER.error("query audit failed for request={}", request);
            throw new BusinessException("query audit failed");
        } finally {
            this.latch.countDown();
        }
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            LOGGER.info("query audit url ={}", url);
+            AuditResponse result = HttpUtils.request(restTemplate,
+                    url,
+                    HttpMethod.GET, null,
+                    null,
+                    AuditResponse.class);
+            LOGGER.info("success to query audit info result={}", result);
+            return CommonBeanUtils.copyListProperties(result.getData(), AuditInfo::new);
+        } catch (Exception e) {
+            LOGGER.info("query audit failed for groupId={}, streamId={}", groupId, streamId, e);
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java:137]: Better to output more information in log like `ip` and `auditId`

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java [hunk 3]
```diff
@@ -412,25 +413,9 @@ public List<AuditVO> listAll(AuditRequest request) throws Exception {
...
-                        vo.setCount(resultSet.getLong("total"));
-                        vo.setDelay(resultSet.getLong("total_delay"));
-                        vo.setSize(resultSet.getLong("total_size"));
-                        auditSet.add(vo);
-                    }
-                    result.add(new AuditVO(auditId, auditName, auditSet, null));
-                }
+                AuditRunnable task = new AuditRunnable(request, auditId, auditName, result, latch, restTemplate,
+                        auditQueryUrl, null, true);
+                this.executor.execute(task);
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java:418]:
```suggestion
                this.executor.execute( new AuditRunnable(request, auditId, auditName, result, latch, restTemplate,
                        auditQueryUrl, null, true));
```

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java:391]: Done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java:137]: Done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java:101]: Done.

**Review by aloyszhang — DISMISSED** [review]
LGTM

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java [hunk 4]
```diff
@@ -147,6 +153,8 @@ public class AuditServiceImpl implements AuditService {
 
     @Value("${audit.query.source}")
     private String auditQuerySource;
+    @Value("${audit.query.url:http://127.0.0.1}")
```
Comment (doleyzi) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java:156]: It is recommended that the default value be consistent with the configuration file and use http://127.0.0.1:10080.

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java:156]: Done.

**Review by doleyzi — APPROVED** [review]
LGTM

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**